en/intro/conventions.rst

```diff
@@ -142,6 +142,18 @@ Now that you've been introduced to CakePHP's fundamentals, you might try a run
 through the :doc:`/tutorials-and-examples/bookmarks/intro` to see how things fit
 together.
 
+Core Naming Conventions
```

Comment:
I think field works there as field is 'generic' across columns & properties, and `find('list')` works on both.